newIDE/app/src/ObjectsList/ObjectRow.js

```diff
@@ -9,6 +9,8 @@ import Clipboard from '../Utils/Clipboard';
 import { CLIPBOARD_KIND } from './ClipboardKind';
 import muiThemeable from 'material-ui/styles/muiThemeable';
 
+const LEFT_MOUSE_BUTTON = 0;
```

Comment:
Always prefer declaring constant that using magical numbers :) Make it easier to understand the code and update it later if we want to refactor this.